src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityTreeBrowser.java

```diff
@@ -363,37 +263,26 @@ public EntityTreeItem makeTreeItemFromQueryResult(EntityQueryResult header,
 		return childItem;
 	}
 
-	public void addResultsToParent(final EntityTreeItem parent,
-			EntityQueryResults results,
-			org.sagebionetworks.repo.model.entity.query.EntityType type,
-			long offset, boolean isExpandable) {
+	public void addResultsToParent(final EntityTreeItem parent,	EntityQueryResults results, long offset, boolean isExpandable) {
 		if (parent == null) {
-			if (type != org.sagebionetworks.repo.model.entity.query.EntityType.folder) {
-				for (EntityQueryResult header : results.getEntities()) {
-					view.appendRootEntityTreeItem(makeTreeItemFromQueryResult(
-							header, true, false));
-				}
-			} else {
-				for (EntityQueryResult header : results.getEntities()) {
-					view.insertRootEntityTreeItem(
-							makeTreeItemFromQueryResult(header, true, true),
-							offset++);
+			for (EntityQueryResult header : results.getEntities()) {
+				String entityType = header.getEntityType();
+				if (entityType.equals("folder")) {
```

Comment:
Could be clarified:
boolean isExpandable = entityType.equals("folder");
view.appendRootEntityTreeItem(makeTreeItemFromQueryResult(header, true, isExpandable));